crates/cache/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+//! Local cache for Wasm modules and static assets pulled from a registry.
```

Comment:
The pattern of `spin_loader` grouping `local` and `bindle` implementations hasn't worked well and I don't think we should expand that to a new crate, even if this PR isn't refactoring `spin_loader`. I would suggest either packing this into `spin-loader` (e.g. `spin_loader::oci::cache`) or - as a step toward where I would hope to refactor all these things - a `spin-oci` crate.

Reply:
Moved the separate cache crate into `spin_loader::oci::cache`.